opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/juniper-graph.properties

```diff
@@ -94,6 +95,29 @@ report.juniper.temp.command=--title="Current Temperature" \
  GPRINT:val3:MAX:"Max  \\: %8.2lf %s\\n"
 
 #####
+##### Juniper SRX reports
+#####
+
+report.srx.spuFlowMonitor.name=Juniper SRX Flows
+report.srx.spuFlowMonitor.columns=jnxJsSPUMonitoringCurrentFlowSession,jnxJsSPUMonitoringMaxFlowSession
+report.srx.spuFlowMonitor.type=jnxJsSPUMonitoringObjectsTable
+report.srx.spuFlowMonitor.propertiesValues=jnxJsSPUMonitoringNodeDescr
```

Comment:
Suggestion: this properties value is not used? Maybe it make sense in the graph title?